### ISSUE TITLE
Tandy 1000 SL/2 slow performance 

### ISSUE THREAD

**Issue by Tape-Worm** [86Box/86Box#3774]
### What happened?

So, I ran Space Quest 3 yesterday and I noticed that the intro took forever to come up (the theme song had finished by the time it came up).  Now, this is on an 8MHz machine and this is an old EGA game, so it should be enough to handle it.

I also noticed that a little program I wrote to draw some stuff on a 320x200 16 colour screen was taking a bit longer than it should have (of course, at the time I assumed it was my horrible 8086 assembly optimization skills).

On a hunch, I tried the Landmark Speed Test application and this is what it came back with:
![8mhz](https://github.com/86Box/86Box/assets/9710137/f540e78d-3ef5-4695-bc1e-05f7e718aeb1)

It's claiming that it's running at 5 MHz.  When I turned it up to 12 MHz, it showed roughly 8 MHz.

I don't expect perfect cycle accuracy, but it seems like that it's running a lot slower than it should be.

### Configuration file

```ini
[Machine]
machine = tandy1000sl2
cpu_family = 8086
cpu_speed = 8000000
cpu_multi = 1
cpu_use_dynarec = 0
time_sync = local
fpu_softfloat = 0
pit_mode = 1
cpu = 1
mem_size = 640

[Video]
video_fullscreen_scale_maximized = 1
gfxcard = internal

[Input devices]
mouse_type = msserial
joystick_type = 2axis_2button
joystick_0_nr = 1
joystick_1_nr = 0
joystick_0_axis_0 = 1
joystick_0_axis_1 = 0
joystick_0_button_0 = 0
joystick_0_button_1 = 1

[Storage controllers]
hdc = xtide

[Floppy and CD-ROM drives]
fdd_01_type = 35_2dd
fdd_02_type = 35_2dd
fdd_02_image_history_01 = 
fdd_01_image_history_01 = 
fdd_01_image_history_02 = 

[WinBox]
window_fixed_res = 960x720

[General]
language = system
vid_renderer = qt_vulkan
window_remember = 1
confirm_save = 0
confirm_exit = 0
confirm_reset = 0
video_filter_method = 0
enable_overscan = 1
video_fullscreen_first = 0
video_fullscreen_scale = 3
scale = 4
vid_resize = 1

[Sound]
fm_driver = nuked

[Monitor #1]
window_coordinates = 323, 105, 1869, 1143

[Monitor #2]
window_coordinates = 0, 0, 0, 0

[Hard disks]
hdd_01_parameters = 17, 15, 963, 0, ide
hdd_01_fn = 
hdd_01_speed = ramdisk
hdd_01_ide_channel = 0:0

[Other peripherals]
isartc_type = p5pak
```

### Operating system

Windows 11

### CPU

AMD Ryzen 7950x

### 86Box version

 v4.0.2.5125

### Build architecture

Windows - x64 (64-bit)

### Build type

- [ ] New recompiler
- [ ] Debug build

### Download source

Manager auto-update

### Additional context

I don't know if the Tandy 1000 HX emulation suffers from the same problem or not, but I'd expect that it would.

**Comment by OBattler** [comment]
This is normal. The 808x emulaton is currently not cycle-accurate and there's likely instructions running to slow. This is going to be fixed in version 4.1 when I finish my 808x rewrite based on MartyPC.

**Comment by OBattler** [comment]
And now I also see that this is being compared to an AT - the AT uses a 286, so the 808x is always going to be slower than it.
